yjit.c

```diff
@@ -871,6 +871,19 @@ rb_yjit_fix_mod_fix(VALUE recv, VALUE obj)
     return rb_fix_mod_fix(recv, obj);
 }
 
+// Return non-zero when `obj` is an array and its last item is a
+// `ruby2_keywords` hash. We don't support this kind of splat.
+size_t
```

Comment:
It shouldn't matter, but any reason why this is `size_t`? `FL_TEST_RAW` seems to return `VALUE`.

Reply:
I just wanted to make sure it zeros out the full register. VALUE indicates returning an object to me so I went with `size_t`. I could change it if you want.